test/e2e/gameserverallocation_test.go

```diff
@@ -60,7 +60,7 @@ func TestCreateFleetAndGameServerAllocate(t *testing.T) {
 			gsa := &allocationv1.GameServerAllocation{
 				Spec: allocationv1.GameServerAllocationSpec{
 					Scheduling: strategy,
-					Required:   allocationv1.GameServerSelector{LabelSelector: metav1.LabelSelector{MatchLabels: map[string]string{agonesv1.FleetNameLabel: flt.ObjectMeta.Name}}},
+					Selectors:  []allocationv1.GameServerSelector{{LabelSelector: metav1.LabelSelector{MatchLabels: map[string]string{agonesv1.FleetNameLabel: flt.ObjectMeta.Name}}}},
```

Comment:
I would recommend we leave in some of our old e2e tests using the deprecated fields, just to make sure the old fields work as expected?

Probably don't need all of them 🤔 but some coverage to ensure required and preferred still work as expected.

Reply:
I said the same thing and I believe that there are some tests that use the old fields being used in both unit tests and e2e tests.

Reply:
There are tests for old fields in allocator_test.go and I just added another one in this file gameserverallocation_test.go.